readthedocs/projects/notifications.py

```diff
@@ -131,5 +132,18 @@
         ),
         type=ERROR,
     ),
+    Message(
+        id=SyncRepositoryLocked.REPOSITORY_LOCKED,
+        header=_("Repository locked"),
+        body=_(
+            textwrap.dedent(
+                """
+                We can't perform the versions/branches synchronize at the moment.
+                There is another sync already running.
+            """
```

Comment:
```suggestion
                There is another sync already running.
                """
```

Reply:
This is managed by dark/black.

Reply:
Don't think black re-formats multiline strings, since they are sensitive to any spacing.